django/core/templates/base.html

```diff
@@ -1,30 +1,102 @@
+
+        <!-- Custom -->
+        <link rel="stylesheet" type="text/css" href="{% static 'css/custom.css' %}">
+        <link rel="stylesheet" type="text/css" href="{% static 'css/custom_small.css' %}">
+        <link rel="stylesheet" type="text/css" href="{% static 'css/custom_large.css' %}">
+
+        <!--Set the favicon-->
+        <link rel="icon" type="image/png" href="{% static 'images/favicon.png' %}">
+    </head>
+
+    <body>
+
+        <!-- jQuery (has to load at start of page, as some templates that extend from this base template include jQuery in them) -->
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.6.0/jquery.min.js"></script>
+
+        <nav>
+
+            <div id="nav-head">
+                <a class="navbar-brand" href="{% url 'general:welcome' %}">
+                    <h1>EUCL<span>CORP</span></h1>
+                </a>
+            </div>
+
+            <ul class="nav-links">
+                <!-- Welcome -->
+                <li><a class="{% if request.path == "/" %}active{% endif %}" href="{% url 'general:welcome' %}"><i class="fas fa-home"></i> Welcome</a></li>
+                <!-- Corpus: Monolingual -->
+                <li><a class="{% if "/corpus/monolingual/" in request.path %}active{% endif %}" href="{% url 'researchdata:monolingual' %}"><i class="fas fa-cube"></i> Monolingual Corpora</a></li>
+                <!-- Corpus: Parallel -->
+                <li><a class="{% if "/corpus/parallel/" in request.path %}active{% endif %}" href="{% url 'researchdata:parallel' %}"><i class="fas fa-cubes"></i> Parallel Corpus</a></li>
+                <!-- About -->
+                <li><a class="{% if request.path == "/about/" %}active{% endif %}" href="{% url 'general:about' %}"><i class="fas fa-info"></i> About</a></li>
+                <!-- Contact -->
+                <li><a class="{% if request.path == "/contact/" %}active{% endif %}" href="{% url 'general:contact' %}"><i class="fas fa-at"></i> Contact</a></li>
+            </ul>
+
+            <div id="nav-base">
+                <a href="https://www.birmingham.ac.uk/" target="_blank">© University of Birmingham {% now "Y" %}</a>
+                <a href="https://www.birmingham.ac.uk/privacy/index.aspx" target="_blank">Privacy</a>
+                <a href="https://www.birmingham.ac.uk/legal/index.aspx" target="_blank">Legal</a>
+                <a href="https://www.birmingham.ac.uk/university/governance/policies-regs/information/index.aspx" target="_blank">Freedom of Information</a>
+                <br>
+                <a href="{% url 'general:cookies' %}">Cookies</a>
+                <a href="https://accessibility.bear.bham.ac.uk/statements-cal.html" target="_blank">Accessibility</a>
+            </div>
+
+        </nav>
+
+        <!-- Main -->
+        <main id="main">
+            <div id="main-content">
+                {% block main %}
+                {% endblock %}
+            </div>
+        </main>
+
+        <!-- JavaScript -->
```

Comment:
Again, I'd remove the unused comments. I guess they were boilerplate?

Reply:
The JavaScript comment is to show below code is JS, which is then split into 3rd party and custom. I've reworded the comments to make this clearer.